src/utils/static-eval.ts

```diff
@@ -327,7 +327,7 @@ const visitors: Record<
 
     return {
       test: node.test,
-      then: thenValue.value,
+      then: thenValue.value === URL ? undefined : thenValue.value,
```

Comment:
This will be confusing for future readers. Can we add a comment explaining why the special case here?